src/DataType/EthD.php

```diff
@@ -70,6 +70,7 @@ class EthD extends EthDataType
         //  bytes<M>: binary type of M bytes, 0 < M <= 32
         // @todo Dynamic sized byte sequence is not yet implemented.
         'bytes' => 'EthD',
+        'bytes32' => 'EthD32',
```

Comment:
Their might be something different, but your commit is exactly in.

https://github.com/digitaldonkey/ethereum-php/blob/6603022d759f32adc9c66e4b8e28b11ab16a5454/src/DataType/EthD.php#L115